daisy/workflow/test.workflow

```diff
@@ -51,30 +51,30 @@
         }
       ]
     },
-    "postinstall stopped": {
+    "postinstall-stopped": {
       "waitForInstancesSignal": [{"name": "postinstall", "stopped": true}]
     },
-    "create image": {
+    "create-image": {
       "createImages": [
         {
           "name": "image-from-disk",
           "sourceDisk": "image"
         }
       ]
     },
-    "sub workflow": {
+    "sub-workflow": {
       "subWorkflow": {
         "path": "./test_sub.workflow"
       }
     }
   },
   "dependencies": {
-    "create disks": [],
```

Comment:
This was mainly for sake of consistency, I figure it's better to enforce it everywhere we think we may use step names in GCE resources in the future. This PR does use subworkflow names as the name for the subworkflow so it was easy to add for everything.